modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -89,8 +91,13 @@ vec4 project_offset_(vec4 offset) {
 // Projecting positions - non-linear projection: lnglats => unit tile [0-1, 0-1]
 //
 vec2 project_mercator_(vec2 lnglat) {
+  float x = lnglat.x;
+  if (project_uWrapCoordinates) {
```

Comment:
Do we have the information at this point to determine whether the antimeridian is in the viewport? If not, we could skip this step, right?